### PR TITLE
chore(docs): Update bug-report.yml

### PR DESCRIPTION
**Pull request by yarikbratashchuk**
## Summary by CodeRabbit

- **New Features**
	- Enhanced bug report template for improved clarity and detail.
	- Added sections for "Network Configuration" and "Development Environment."

- **Improvements**
	- Updated prompts for version information and system specifications.
	- Refined sections for "Steps to Reproduce," "Expected Result," and "Actual Result" for better user guidance.
	- Clarified requests for relevant logs and additional information.

### CODE REVIEW

**Review by Manav-Aggarwal — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -5,76 +5,129 @@ body:
...
+      description: Please provide versions of all relevant components
+      placeholder: |
+        Git SHA: 
+        Rollkit Version:
+        Go-DA Version:
+        Artela-Rollkit Version:
+        Celestia Version:
+        Sequencer Version:
+        Go Version:
+        Protobuf Version:
```
Comment (MSevey) [inline .github/ISSUE_TEMPLATE/bug-report.yml:22]:
```suggestion
        Rollkit:
        Go-DA:
        DA Implementation:
        Go-Sequencing:
        Sequencing Implementation:
        Go-Execution:
        Execution Implementation:
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -5,76 +5,129 @@ body:
...
     attributes:
-      label: Installed tools
-      description: e.g. docker, makefiles, etc.
+      label: Development Environment
+      description: List all relevant tools and their versions
+      placeholder: |
+        Docker Version:
+        Docker Compose Version:
+        Make Version:
+        Node Version:
```
Comment (MSevey) [inline .github/ISSUE_TEMPLATE/bug-report.yml:63]:
```suggestion
        NodeJS Version (if applicable):
        Go Version:
        Protobuf Verion:
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -5,76 +5,129 @@ body:
...
+    id: config
     attributes:
-      label: Others
-      description: >
-        e.g. flag options, celestia config file changes, resources
-        limitation(like cpu, ram limit, swap etc.)
+      label: Configuration Details
+      description: Any modifications to default configurations
+      placeholder: |
+        Celestia Config Changes:
```
Comment (MSevey) [inline .github/ISSUE_TEMPLATE/bug-report.yml:72]:
```suggestion
        DA Implementation Config:
        Sequencer Implementation Config:
        Execution Implementation Config:
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -5,76 +5,129 @@ body:
...
-      description: >
-        Please copy and paste any relevant log(max 20 lines) output. This will
-        be automatically formatted into code, so no need for backticks. Or paste
-        gists, pastebins links here
+      label: Relevant Logs
+      description: |
+        Please provide relevant logs from different components:
+        - Node logs
+        - Rollkit logs
+        - Celestia logs
```
Comment (MSevey) [inline .github/ISSUE_TEMPLATE/bug-report.yml:118]:
```suggestion
        - DA logs
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -5,76 +5,129 @@ body:
...
+        - Sequencer logs
```
Comment (MSevey) [inline .github/ISSUE_TEMPLATE/bug-report.yml:119]:
```suggestion
        - Sequencer logs
        - Execution logs
```

**Review by MSevey — CHANGES_REQUESTED** [review]
Let's make this more generic and not specific to celestia and artela

**Review by MSevey — APPROVED** [review]

**Pull request merged**